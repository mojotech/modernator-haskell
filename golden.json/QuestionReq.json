{
    "seed": -6.599569919912392651e18,
    "samples": [
        {
            "question": "`im§è~¤<\n\u0014?y!»{pÁî]\u001feër\u001f"
        },
        {
            "question": "Aö\u0012\u001ct(^ßý\u001dMÍ\u001c¦ë\u0011þÔ\u0018MäóV\u00089@&Ë"
        },
        {
            "question": "þn¯û$M<\u0015g»6L¾\u000fÝB,|&"
        },
        {
            "question": "(\u0018\u0006w\u0013/J"
        },
        {
            "question": "$,\u000c+-;0¡6\u0014u¾Ê¯\u0017Q4ÞGgy"
        },
        {
            "question": "\u000e5VÌ\u00111,\u001cÜ\u0012ù&Kø:\u001e9*Guoo\u001d\r_\u0003"
        },
        {
            "question": "áÍ0²ÿ\"HBJkµ"
        },
        {
            "question": "\u000eá\u0017/\u000efgbZ\"\u001e@P[É/8SízG+"
        },
        {
            "question": "}\u0001Q\u001cÄSß:øMH\u0015zÃ\u0015\u001e"
        },
        {
            "question": ""
        },
        {
            "question": "ÒNEw·3[\u001b\u001824\u0004)ss_}\u0015\u0011"
        },
        {
            "question": "F~×ÑY¿T)gJxFv+~èj\u000cð<²W[\u001eËJ"
        },
        {
            "question": "OJÍ ÛCR#L\u0003oZ²d\u0011"
        },
        {
            "question": ""
        },
        {
            "question": "\u0018@\u001e\u0000\u001f1$/2\u001dÈ?\u000e%\u001f³X"
        },
        {
            "question": "\u0011\u0013lS\u000c;g\u0013YB³KhMT£_F"
        },
        {
            "question": "L\r"
        },
        {
            "question": "¹4éê\u001dFÉ/^|dã\u0003©*\u000f(4S\u0019A"
        },
        {
            "question": "$¥,½8"
        },
        {
            "question": "ù9z\u0010*.S\u0017\rM]\u0018"
        },
        {
            "question": "\u0014\noæf/\u0010ø"
        },
        {
            "question": "~ënatêP1%$\u0019dº¬"
        },
        {
            "question": "v\u0007BH$YH3\u0012L$~ÚC+å2\u0018hëQ\u001a\u0008a3"
        },
        {
            "question": "v ÜÒ$"
        },
        {
            "question": "°\u0003Y\u000fY$\u0006g&^\\ziÂ;ó+=^"
        },
        {
            "question": "C3\u0016'\u0017Yoðn0\u0012/"
        },
        {
            "question": "~9}A"
        },
        {
            "question": "PuþO"
        },
        {
            "question": "\u0007ß?ÀWv\u0007"
        },
        {
            "question": "\u000e\u0004Û÷oÍZrwmÁZu\u0002_sÈ+[DÒ+"
        },
        {
            "question": "\u0019g\u00038\\\u0008)Á\u0011\u0007 rQ1W\u0000ÃFIM21"
        },
        {
            "question": "J\u001aÅf"
        },
        {
            "question": "$j\nf}j7&|gL\u000e\u0015B=\u0007Ul¥BZlÇ"
        },
        {
            "question": "u%"
        },
        {
            "question": " aOë<\u0013i&q\u0005H%\u00116`t.I­"
        },
        {
            "question": "A0sa\u0002ie7K>\u0019íMµ²j?\u001eDG%\u0012YÞt\u0013\u000e"
        },
        {
            "question": ")¨`Ét$*"
        },
        {
            "question": "|'|JÙ\u000c?veL\u00196z×F\u000eBë2MÖÜrð"
        },
        {
            "question": ".dpT;\u0007Áit?»>iA\u0014\u001aÝ-\u0013C"
        },
        {
            "question": "\u000fTj)½<[aÓ \u000c:,L¢ø"
        },
        {
            "question": "\u001e\u001a2I{\u000b@OcF\u001b\u0002%uì?\u001e\t»>oÌ[\nÁ\u0011/"
        },
        {
            "question": "@I\u0014|»\u001dÌLn"
        },
        {
            "question": "\u001ec^(\u0014k\u0017"
        },
        {
            "question": "\nÿó\u0008^\t¶ 5WJA\u0017dFv>Gfy²Ð>\u0003Vl"
        },
        {
            "question": "#Ô\u0013è}v,f\u0005\u001e\u000b+¥_"
        },
        {
            "question": "WyT-q4ëTä!¬\u0006\u0014"
        },
        {
            "question": "\u001a~Z;\tB\u0015m\u0008\u0017IøÀ\u0003OGBu"
        },
        {
            "question": ""
        },
        {
            "question": "lT^\u0005FÃId'\u0015K9+"
        },
        {
            "question": "&¡%3g"
        },
        {
            "question": "`\u001czZFPGº \u001eRÕïÕ\u0005\u0006^\u0017xø"
        },
        {
            "question": "-\\&xÐ,Yø\u000b~\r\tÁc\u0008PÂJ²meX\u001430\\q\""
        },
        {
            "question": "ÿR3J:{!\u0007(ÌNØd\u000e#@OódlèyàO\u0003\u001e"
        },
        {
            "question": "Þ\u0008\u001c\u0015%Ã\u001fM\u001f\u000bN"
        },
        {
            "question": "B\u0011¬Ò.r7æ\u0011mrá\u000c\u001el"
        },
        {
            "question": "%2X3-NSÖuMLRhj×_\u0015'@9Ù\t*fº'\u0016"
        },
        {
            "question": "=N&\\Uï\u000e­ÚÚ\u0006P8wZ\u00010îr³RÝe"
        },
        {
            "question": "ÞÄú4J@.\u001cÇÆÂCcu\"þ\u001b'2N"
        },
        {
            "question": "7J@Ö¬4 "
        },
        {
            "question": "={t8"
        },
        {
            "question": "NP\u0017\u0019*}[&Þ"
        },
        {
            "question": "\u0013?E\\I;_J@\u0005Aý\u0004\u001dXÍ])Y\u0016r?&"
        },
        {
            "question": " _c~ XQk_\u0001Éns&"
        },
        {
            "question": "\u001d\" £\u000bß\u0015Z\\\u000bj\u000cq/JK§?\u0004=\u000eg\u0005É"
        },
        {
            "question": "q\u0014\r>\u0019gº"
        },
        {
            "question": "}!\u0017RxÔo\u0002k\riÎr@85Ýï\u000bsg7Æ\u0004LW"
        },
        {
            "question": "\n\u00011+\"W!aiøÿÚ+ w"
        },
        {
            "question": "]J\u0017wK\ria\n5gk"
        },
        {
            "question": "+_inP&\u0006e-n\u001d\u001e\u0012MA$ b­2\t\u0019"
        },
        {
            "question": "Ye\u000f"
        },
        {
            "question": "ÒÛÖ)\u001as*21M?\u0006Yv¡d/q)"
        },
        {
            "question": "U/O'\u0004\nØNb\"\u0001±Z\u0012jÚa\u001fk"
        },
        {
            "question": "?ß"
        },
        {
            "question": ".\u0013f\u0000BT9Z\u0018\u0014I¬¶»`QA0l"
        },
        {
            "question": "+H$\u0019\u000c[\u00037\u0010ß^Gt»'1ÖMr\u0013r\u000f"
        },
        {
            "question": "HX½Ybk'\u0006F\t»y2e!}\u0006mm6"
        },
        {
            "question": ""
        },
        {
            "question": "7"
        },
        {
            "question": "33:jQtRà}´"
        },
        {
            "question": "WTRDYo~wQRN3¹Xìe\u00024t&'Øasg"
        },
        {
            "question": "N\\{\u0013!Ô7¥\u001f¯=p4;z@à\u001ae"
        },
        {
            "question": "D"
        },
        {
            "question": "Rû"
        },
        {
            "question": "{\u0012o`Ï\u000fT5eSiT\u0018)í8, \u0004\u0000"
        },
        {
            "question": "Ë\rg\nRA7ÞH»lbL5*5i\u0010¹k"
        },
        {
            "question": "(áÂ al"
        },
        {
            "question": "p<9¨\u0004#ô6\u0005>Çm=Õ\u0010\u001f}lM"
        },
        {
            "question": "P[7Æð=Vj1g¾"
        },
        {
            "question": "7wì~EO*\u001c%x*WBZ"
        },
        {
            "question": "\u000crN\u0013\u000bi\u0011[6w\u0004vàTMm»\u001a"
        },
        {
            "question": ""
        },
        {
            "question": "]m\u001d^~°ì;ãI: \u001e"
        },
        {
            "question": "pô)KZX¨\u0011\u0011Ä=r(k<?Zrå;,Y"
        },
        {
            "question": "<@\u0001H"
        },
        {
            "question": "\tÊÝ¼Pyz}\t\u001e_/¸Ú"
        },
        {
            "question": "bGc\u000f31\u0003sP=SJ\u0019bc\u0002Ùs\u0008i"
        },
        {
            "question": "\u0013FR6\u000b`òb\r°V¹\tÍ\u0010=Y"
        },
        {
            "question": "4MÒGi\u000097úZ5"
        },
        {
            "question": "ò'\u0004.\rZÓIx|h°N(,L­\u00061"
        },
        {
            "question": "¦\u0004Z;\u0005a\rÊQê\\©\u0005Vñ\u0015@:LàÍ9"
        },
        {
            "question": "\u0017 '$"
        },
        {
            "question": "j\u000f©ô\\7\u0011m\u000c*\u0008N|$<2E\u001ai\u000c[\u0013*n£"
        },
        {
            "question": "\u001bÉV´2Xq7uÎrDu)\u0013\"_{\u001cU\\R\u000c\u0005 -\u000b"
        },
        {
            "question": "\u0016r^as0"
        },
        {
            "question": "ókú\u0011hyZö\u001f9_FxwD{|"
        },
        {
            "question": "Ò?zsJgCrÛ@8^¡"
        },
        {
            "question": "\u0007yaV6\u001f* }']Ì<Y\"XÅ\u0018Ò¦0@úÆcKµ\u001f=\u001a"
        },
        {
            "question": "\u0019lÖ\u0010µ¤`Bnb\u0013k"
        },
        {
            "question": "\u0002$9"
        },
        {
            "question": "F{,÷¶kNE´ÔâCH7w\u0006QeJm"
        },
        {
            "question": "u\u0007DR;À\u0005öd'kwÑ%Ã>]\u000e\u0019\u0002W\u0006þ"
        },
        {
            "question": "K\u001a#Õ\u0004º\u0003\u000b\u0006!#%|s"
        },
        {
            "question": "$\r;EñÓOl·"
        },
        {
            "question": "i6\r, ßX|l$[j_A\u0010[sTsO"
        },
        {
            "question": "o\t²\u001d)¥[ð\\"
        },
        {
            "question": "m1\u0012÷s`VÂ^)S\u0002[4'[\u0010¹\u001c)"
        },
        {
            "question": "\":\u001c·\u000e\u000b\t]\u0019,"
        },
        {
            "question": "p2Ry$vx©\u0015\u0004&$(Sèu\u00112\"ó"
        },
        {
            "question": "]]&@"
        },
        {
            "question": "U¢1Âbi6Ù"
        },
        {
            "question": "yX¸z\u000eH\r"
        },
        {
            "question": "G"
        },
        {
            "question": "&¸>rSF²kD5;a{1\u0013\u0015\u0007()}j#Om"
        },
        {
            "question": "]Ös4®\u0014,[¤ô]\"×¢}|}|\u000cQN\u000cl9U1Ê"
        },
        {
            "question": "çG.£v}în\u0016\u0018Ndj_;¬@\u0015¿x+ºQÕR6 \u0005"
        },
        {
            "question": "Kk'\n\u000bf\u001dÜ|X.\u0005*\u0007tL\u0019&_ «Ð;\u001eL"
        },
        {
            "question": "ÿ\u0000*\u0000\u001bL"
        },
        {
            "question": "q9^p*\u0013\u0011\u0010\u0000"
        },
        {
            "question": ""
        },
        {
            "question": "\u001chJT]W¹O\u0003\u001dæk"
        },
        {
            "question": "Ký\u0008÷o0[\u0016é\u001a|\u001c\u001b)¹"
        },
        {
            "question": "\u0018\u001b\u001e\u000fx\u0010=k`t\u0014TwF#e'>íëöáÃ&!-W\u0019"
        },
        {
            "question": "FhÄöì@«pG"
        },
        {
            "question": "S\u001d"
        },
        {
            "question": "ë@\u0017àf\"\u0004r/\u0018ô%ca1râß2'"
        },
        {
            "question": "á36q\u000f\u000cÜñRq)"
        },
        {
            "question": "RD´]\u001dnöjêÉ1M\u00180w^s\"/h®ô¶!d\u0011-V"
        },
        {
            "question": "Ô¶"
        },
        {
            "question": "gC4l=&\u0014l[\u0011þ¡M%@"
        },
        {
            "question": "I²+:8U>\u0004ä&Ti8\u0018õ³w\u0013Ë}"
        },
        {
            "question": "\u000f\u0016º>"
        },
        {
            "question": "j26pH]óRD/-|Â6,¯"
        },
        {
            "question": "sÙg]u\nwUl)"
        },
        {
            "question": "\u001c\u0018ýñ\u0015BLµ\u001a\u0019*þ\\\u0004,,T{\u0017\u0013l \u0000E\u0006\t"
        },
        {
            "question": "¦J8k'\u0010a:RWP\u001b+\u0013£#3w<Òp\\\u0007\u001bX"
        },
        {
            "question": "z¯á\u0010;Ë¢1=\u001f\u001a"
        },
        {
            "question": "\u0015S¼"
        },
        {
            "question": "\tQ\u0012f4"
        },
        {
            "question": "Ïz\u0007\u001b¨x"
        },
        {
            "question": "\u0006\u001dzv§\u001c ¸!\u001bjnY0!46Õ\u0019ú\t,^êd"
        },
        {
            "question": "ôÎlË-=`<{ºmõî4WmI"
        },
        {
            "question": "?Þ[5\u0019Ù\u00132'aÉïÕ\u000cGá¿tñyk\u0011\u000f9pòM\n"
        },
        {
            "question": "¾â\u0018T¨\u0015\u0012ú\u0002\u0015V#Vz{Ê<\u0018qª2N?<[&"
        },
        {
            "question": "0Ä\u001c;ÄxÖæ\u0004PMù"
        },
        {
            "question": "|\u0018\u001dl\u000cb©n\u0001v\u0008IÕ@"
        },
        {
            "question": "\u000c\\@\u000eU/]\u001a"
        },
        {
            "question": ".Å\rL\\*x9kIl\""
        },
        {
            "question": "ÁyUZ¢rë×¥ã"
        },
        {
            "question": "\u0007Y7\u0011ÊÕ\u0012B\u0019I\u0001_×gªfY\u000eø\u000eaá\u0012§ cG~"
        },
        {
            "question": "¯"
        },
        {
            "question": "\u001fo\u0018ª=ZÊ\u0000Ó\u0000&"
        },
        {
            "question": ")o^\u0001-«"
        },
        {
            "question": "uÙT\r4ÒÌ\u0001\u0003\u0018jw9*"
        },
        {
            "question": "`"
        },
        {
            "question": "\u0000u5\u0004'\\Ha\u0017U"
        },
        {
            "question": "Mq¾dâ6\u0011Ê\u000b\u001f8\u0010\\Ç,\u0008\u0019F#"
        },
        {
            "question": "\\Xn0UJ?uæ\u0017V\u001f\u0004c'L$Rfq\u0012+ B\r\u000cz"
        },
        {
            "question": "ÿ9dj±lgt(ÞDc0\u00177o\u0005\u0011\u0004GY"
        },
        {
            "question": "Ro\\"
        },
        {
            "question": "¸\n|{7Un\u0018b¦\u000eæÆk(³>Àh\u0017çtm"
        },
        {
            "question": "\u0016\u0019|ßSF\u0008Z\u0014B\u0006ØÿT=b\u000bQ_çý.vJ$"
        },
        {
            "question": "j^¸"
        },
        {
            "question": "\u0014#w!ÉpF\u001d\u000cª\u0013"
        },
        {
            "question": "S¤;\u0008jR£\u00185;Cã¤7$\ne\u001aq\u00000zçKn'×F3"
        },
        {
            "question": "È5Di÷W\u001eÓwR;r- ¿xl\t\\©r\u0017\u0017æ"
        },
        {
            "question": "³"
        },
        {
            "question": "OfN\u001dûrq]"
        },
        {
            "question": "&\u000fq"
        },
        {
            "question": "\u0008ÀÃJ½"
        },
        {
            "question": "t\u000eMïG'ëÃÐ\u0003Qv"
        },
        {
            "question": "\rbË?{9\u000c~.°_\u0015XrD>\"æKÈ$bõ"
        },
        {
            "question": "h[\u001bfg\\\u0003\u0010?Xz\u001bC\u001cj\u001f|u"
        },
        {
            "question": "qe"
        },
        {
            "question": "?ë"
        },
        {
            "question": "> \u000c\u001b^T\u0018n<\u000fWk.û>cp\u0002wMgR\u0007dÙ3{x¢"
        },
        {
            "question": "ëóLº\u00194\u001f"
        },
        {
            "question": "eXY3l\u0014t%f"
        },
        {
            "question": "\u0010 ./\u0015Xi{IÈ°"
        },
        {
            "question": "1iÿä\u0019;Q2\u000c\\h\u0017\u0007"
        },
        {
            "question": "ò¬FÃ\u0012%(."
        },
        {
            "question": ""
        },
        {
            "question": "eY\u000cv'Q]wp'6&Ñ\u0004~*`j\u001au2,'"
        },
        {
            "question": "`ª\u0004¡\u0004í\u0013C^8)Åx\u0001G@ãX\u0011fU\u0001"
        },
        {
            "question": "bäì²hASpJf\u0014e2"
        },
        {
            "question": "¹MHnYþÛuGm.x\u0014\tn|!£)]_\u0004~\u000b"
        },
        {
            "question": "5<²\"*ó\u001dÔ?"
        },
        {
            "question": "Î¿f"
        },
        {
            "question": "xpçx2­V\u0005\t¾D¥qVÿ¤\u000eS\u000b%$ðeqx;{o"
        },
        {
            "question": "aFµ`\u0007iaÏ\u000e¯ú6v_{L\u001c"
        },
        {
            "question": "n"
        }
    ]
}