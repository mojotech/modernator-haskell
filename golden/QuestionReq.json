{
    "seed": 1981256103533824043,
    "samples": [
        {
            "question": "´vÄ Ð\u0018>yH§\n"
        },
        {
            "question": "kPø`ÝuP\\­\u001d{R\u000b°,hw._\u0008\u0018}ÍYÂ3î7l"
        },
        {
            "question": "d\u001bq|$^\u0008 tj2\u000er9#EdJ!\u0014$Éj"
        },
        {
            "question": "GHcI@\táIÃB\u0011C8ç\u0003#jlnf\t\u000b2?\u0015Ýro"
        },
        {
            "question": "Ñ\\n4j8E8%e"
        }
    ]
}